content/en/continuous_integration/setup_tests/ruby.md

```diff
@@ -44,76 +41,92 @@ To install the Ruby tracer:
 
 {{< code-block lang="ruby" >}}
 require 'cucumber'
 require 'datadog/ci'
 
-# Configure default Cucumber integration
 Datadog.configure do |c|
   c.ci_mode.enabled = true
-  c.use :cucumber, options
-end
-
-# Attach tags from scenario to active span
-Around do |scenario, block|
-  active_span = Datadog.configuration[:cucumber][:tracer].active_span
-  unless active_span.nil?
-    scenario.tags.filter { |tag| tag.include? ':' }.each do |tag|
-      active_span.set_tag(*tag.name.split(':', 2))
-    end
-  end
-  block.call
+  c.service = 'my-ruby-app'  # Name of the service or library under test
+  c.use :cucumber
 end
 {{< /code-block >}}
 
-Where `options` is an optional `Hash` that accepts the following parameters:
+Run your tests as you normally do, specifying the environment where test are being run (e.g. `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable. For example:
 
-| Key | Description | Default |
-| --- | ----------- | ------- |
-| `enabled` | Specifies whether Cucumber tests should be traced. Useful for temporarily disabling tracing. `true` or `false` | `true` |
-| `service_name` | Specifies the service name used for `cucumber` instrumentation. | `'cucumber'` |
-| `operation_name` | Specifies an operation name used for `cucumber` instrumentation. Useful if you want to rename automatic trace metrics in the form `trace.#{operation_name}.errors`. | `'cucumber.test'` |
+{{< code-block lang="bash" >}}
+DD_ENV=ci bundle exec rake cucumber
+{{< /code-block >}}
 
-See also the [Ruby trace library documentation for Cucumber][3].
 
 ### RSpec
 
 The RSpec integration traces all executions of example groups and examples when using the `rspec` test framework.
 
 To activate your integration, add this to the `spec_helper.rb` file:
 
-{{< code-block lang="ruby" >}}
+{{< code-block lang="ruby" filename="spec_helper.rb" >}}
 require 'rspec'
 require 'datadog/ci'
 
-# Configure default RSpec integration
 Datadog.configure do |c|
   c.ci_mode.enabled = true
-  c.use :rspec, options
+  c.service = 'my-ruby-app'  # Name of the service or library under test
+  c.use :rspec
 end
 {{< /code-block >}}
 
-Where `options` is an optional `Hash` that accepts the following parameters:
+Run your tests as you normally do, specifying the environment where test are being run (e.g. `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable. For example:
+
+{{< code-block lang="bash" >}}
+DD_ENV=ci bundle exec rake spec
+{{< /code-block >}}
+
+
+## Additional configuration settings
+
+The following is a list of the most important configuration settings that can be used with the tracer, either in code by using a `Datadog.configure` block, or using environment variables:
+
+`service`
+: Name of the service or library under test.<br/>
+**Environment variable**: `DD_SERVICE`<br/>
+**Default**: `"pytest"`<br/>
```

Comment:
Does ruby also have pytest? Looks like this may be a wrong default service for ruby.

Reply:
Good catch, my bad